connectors/tests/ftest.sh

```diff
@@ -44,17 +44,17 @@ if [[ $PERF8 == "yes" ]]
 then
     if [[ $PLATFORM == "darwin" ]]
     then
-      $PERF8_BIN --refresh-rate $REFRESH_RATE -t $ROOT_DIR/perf8-report-$NAME --asyncstats --memray --psutil -c $ELASTIC_INGEST --one-sync --sync-now --debug
+      $PERF8_BIN --refresh-rate $REFRESH_RATE -t $ROOT_DIR/perf8-report-$NAME --asyncstats --memray --psutil -c $ELASTIC_INGEST --debug
     else
-      $PERF8_BIN --refresh-rate $REFRESH_RATE -t $ROOT_DIR/perf8-report-$NAME --asyncstats --memray --psutil -c $ELASTIC_INGEST --one-sync --sync-now --debug
+      $PERF8_BIN --refresh-rate $REFRESH_RATE -t $ROOT_DIR/perf8-report-$NAME --asyncstats --memray --psutil -c $ELASTIC_INGEST --debug
     fi
 else
-    $ELASTIC_INGEST --one-sync --sync-now --debug
+    $ELASTIC_INGEST --debug
 fi
 
 $PYTHON fixture.py --name $NAME --action remove
 
-$ELASTIC_INGEST --one-sync --sync-now --debug
+$ELASTIC_INGEST --debug
```

Comment:
How will `ftest` finish its run now after the deletion of flags?

Reply:
I'm making an experiment:
1. make the sync job run in the background
2. start a monitoring process and periodically check `last_synced` of the connector
3. if `last_synced` changes or it reaches time out, kill the sync job process.

Reply:
I see, the monitoring process is going to be in another PR? I don't see the logic here :(

Reply:
It's here: https://github.com/elastic/connectors-python/pull/285/commits/4f405cadea203a556d1ddaff48bc87b2982ea740

cc @tarekziade